Automattic/sensei
# Remove typography font family settings from blocks in email editor

Fixes https://github.com/Automattic/sensei/issues/6525

### Changes proposed in this Pull Request

* Removed the font-family option from blocks loaded inside email editor.

### Testing instructions

- Enable `sensei_email` post type
- Create a post of type `sensei_email`
- Add a paragraph (or any other available block in it)
- Make sure the `font-family` option does not appear in the `typography` settings for that block.

### Screenshot / Video
![image](https://user-images.githubusercontent.com/6820724/219314272-4a08961b-6b60-4c15-82ec-f9602dc79dd7.png)
